misk/src/main/kotlin/misk/web/MiskWebFormBuilder.kt

```diff
@@ -35,34 +37,35 @@ class MiskWebFormBuilder {
     }
 
     val typesMap = mutableMapOf<String, Type>()
-    val stack = LinkedList<Class<*>>()
-    stack.push(requestClass.java)
+    val stack = LinkedList<KClass<*>>()
+    stack.push(requestClass)
 
     while (stack.isNotEmpty()) {
       val clazz = stack.pop()
 
       // No need to re-process a given type.
       // This acts as the visited set of our type graph traversal.
-      if (typesMap.containsKey(clazz.canonicalName!!)) {
+      if (typesMap.containsKey(clazz.java.canonicalName!!)) {
         continue
       }
 
       val fields = mutableListOf<Field>()
 
-      for (property in clazz.declaredFields) {
+      for (property in clazz.memberProperties) {
```

Comment:
what's the difference with memberProperties? Doese this capture all the same fields as declaredFields?